test/fixtures/mocha-hooks.js

```diff
@@ -3,12 +3,20 @@
 const { promises: fs } = require("fs");
 const paths = require("../utils/paths");
 
+process.env.NODE_ENV = "test";
+
 /**
  * Re-create the .tmp directories before each test
  */
 beforeEach("clean the .tmp directory", async () => {
   // Delete the .tmp directory, if it exists
-  await fs.rmdir(paths.tmp, { recursive: true });
+  try {
+    await fs.rm(paths.tmp, { recursive: true });
+  }
+  catch (err) {
+    if (err.code === "ENOENT") return;
+    throw err;
+  }
```

Comment:
`fs.rm` has a `force` option to replicate `rm -rf`: https://nodejs.org/docs/latest-v16.x/api/fs.html#fspromisesrmpath-options

```suggestion
  await fs.rm(paths.tmp, { recursive: true, force: true });
```